src/analysis/modals/ComputeModal/GcpComputeModal/GcpComputeImageSection.test.ts

```diff
@@ -81,7 +63,6 @@ describe('GcpComputeImageSection', () => {
   beforeEach(() => {
     // Arrange
     asMockedFn(useComputeImages).mockReturnValue(defaultComputeImageStore);
-    asMockedFn(Ajax).mockReturnValue(mockAjax as AjaxContract);
```

Comment:
Removed.  I couldn't find any usage of Ajax data calls in the code under test that is not already covered by mocking other dependencies.... I am guessing this is a vestige of an earlier version of the code under test.